src/VersionInformation/index.jsx

```diff
@@ -1,13 +1,13 @@
-import React, { useMemo } from 'react';
-import { gtaModel } from '../constants/gtaModel';
+import { useMemo } from 'react';
 import { useParams } from 'react-router';
```

Comment:
Import from react-router-dom. Make sure everywhere there is something imported from react-router, import it from react-router-dom